src/containers/career.jsx

```diff
@@ -18,14 +19,36 @@ class Career extends Component {
     super(props);
 
     this.state = {
-      currentIndex: 0
+      currentIndex: 0,
+      careers: []
     };
     this.addToLocal = this.addToLocal.bind(this);
     this.nextCareer = this.nextCareer.bind(this);
     this.flipCard = this.flipCard.bind(this);
     this.flipCardBack = this.flipCardBack.bind(this);
   }
 
+  componentWillMount() {
+    if (localStorage.liked) {
+      const Liked = JSON.parse(localStorage.liked);
+      const Data = this.props.careers;
+      const filteredData = _.differenceBy(Data, Liked, 'title');
```

Comment:
I think this would have worked without having to import a whole package:
```js
Data.filter((career) => {
	for(var i=0, len = Liked.length; i<len; i++){
		if (career.title === Liked[i].title) {
			return false
		}
	}
	return true
})
```

Reply:
and then also `filteredData` could be `notLikedCareers` as it doesn't describe how it's filter. this is better naming than the other 2 tho